tests/components/opentherm_gw/test_config_flow.py

```diff
@@ -182,13 +182,13 @@
     )
     entry.add_to_hass(hass)
 
-    result = await hass.config_entries.options.flow.async_init(
+    result = await hass.config_entries.async_init(
```

Comment:
This change doesn't look correct. We want to test the options flow, not the config entries flow.

Reply:
Yep, was supposed to be `hass.config_entries.options.async_init(` but fat fingered something. Fixed.